src/Chapter1.hs

```diff
@@ -653,8 +670,10 @@ You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
 
-firstDigit n = error "firstDigit: Not implemented!"
-
+firstDigit :: Int -> Int
+firstDigit n | n < 0 = firstDigit (-n)
+firstDigit n | n < 10 = n
+firstDigit n = firstDigit $ n `div` 10
```

Comment:
You can use all guards under one function name. As I see you use `firstDigit n` everywhere, so it makes sense to make it shorter and not repeat that:
```suggestion
firstDigit n 
    | n < 0 = firstDigit (-n)
    | n < 10 = n
    | otherwise = firstDigit $ n `div` 10
```